app/views/layouts/_primary_navigation.html.haml

```diff
@@ -15,7 +15,10 @@
           %ul.govuk-list
             - if current_user.persona.is_a?(SuperAdmin)
               %li
-                = govuk_link_to t('.sidekiq_console'), sidekiq_web_path
+                = govuk_link_to t('.sidekiq_console'), sidekiq_web_path, class: cp(sidekiq_web_path)
```

Comment:
Helper `cp()` isn't new to this PR, is it possible to rename this helper to be more descriptive within this PR, if not, let's create a new ticket to handle it.